mkt/webpay/tests/test_webpay_jwt.py

```diff
@@ -125,6 +145,30 @@ def test_webapp_product(self):
         eq_(product_data['addon_id'], self.product.addon().pk)
         eq_(product_data['application_size'], self.product.application_size())
 
+    @mock.patch.object(settings, 'AMO_LANGUAGES', ('en-US', 'es', 'fr'))
```

Comment:
You can use [`django.test.utils.override_settings`](https://docs.djangoproject.com/en/1.6/topics/testing/tools/#django.test.utils.override_settings) instead of `mock`. Looks like it's already imported.

```py
@override_settings(AMO_LANGUAGES=('en-US', 'es', 'fr'))
def test_localized_properties(self):
  # ...